assets/templates/oauth1/complete.html

```diff
@@ -54,6 +55,26 @@
 				margin-right: 4px;
 			}
 		</style>
+		<script>
+			const handleDisconnect = () => {
+				// Splitting the cookies string on the basis of cookie name and then popping out the first value.
+				const token = ('; ' + document.cookie).split('; MMCSRF=').pop().split(';')[0];
+
+				fetch("{{ .RevokeURL }}", {
+					method: "POST",
+					headers: {'X-CSRF-Token': token},
+				}).then(res => {
+					const element = document.body;
+					if(!res.ok) {
+						element.style.margin = "10px";
+      				} else {
+						window.close();
+					}
+
+					return res.text().then(text => element.innerHTML = text);
```

Comment:
Do we expect an HTML rich response? If not, can we use `innerText`?What is the contents of the `text`?

Reply:
Yeah, it's going to be HTML type response. This will basically contain the HTML message saying `Account has been disconnected` OR the error message.

Reply:
Thanks for the explanation!